data/config/ndef-qa.json

```diff
@@ -57,6 +57,7 @@
           "name": "ImmPort",
           "body": "ImmPort is funded by the NIH, NIAID, and DAIT in support of the NIH mission to share data with the public. Data is provided by NIH-funded programs and other research organizations.",
           "external_link": "https://www.immport.org/home",
+          "external_link_text": "Visit Website",
```

Comment:
if we do this then I would change the similiar field name to `internal_link_text` for internal button, right now it is called `label`, jsut for consistence

Reply:
cool, will change this accordingly